cluster/addons/dns/kube2sky/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
```

Comment:
How about using [progrium/busybox](https://github.com/progrium/busybox) instead? We use that for cadvisor and heapster and it is very helpful.